storage/tests/integration/signed_url_test.py

```diff
@@ -26,9 +25,7 @@ async def test_gcs_signed_url(bucket_name, creds, data):
         bucket = Bucket(storage, bucket_name)
         blob = await bucket.get_blob(object_name, session=session)
 
-        iam_client = IamClient(service_file=creds, session=session)
-
-        signed_url = await blob.get_signed_url(60, iam_client=iam_client)
+        signed_url = await blob.get_signed_url(60)
```

Comment:
errp ... didja wanna mock any of this?

Reply:
Nope, not for an integration test! Generally, you want to mock things in unit tests to ensure you test only your discrete component, but integration tests should not have anything about your app itself mocked. Since this method no longer requires a connection to GCP, there's nothing to mock there!

Also, this repo somewhat misnamed our integration test folders -- these tend to actually be smoke tests where we're literally hitting the GCP endpoints, since for a library whose only job is to access an API, that middle ground of integration testing where the API is mocked doesn't really test you all that much.